src/Uno.UI/UI/Xaml/Controls/Grid/Grid.cs

```diff
@@ -1355,6 +1348,14 @@ protected override XSIZEF ArrangeOverride(XSIZEF finalSize)
 				}
 				else
 				{
+					// UNO NRE FIX
```

Comment:
Is this comment still needed ?

Reply:
yes, this is still an Uno-specific workaround. (There's presumably some underlying difference in the Measure+Arrange logic that makes it necessary.)